sdk/core/azure-core/azure/core/pipeline/policies/_distributed_tracing.py

```diff
@@ -61,10 +64,7 @@ def _default_network_span_namer(http_request: HTTPRequestType) -> str:
     :returns: The string to use as network span name
     :rtype: str
     """
-    path = urllib.parse.urlparse(http_request.url).path
-    if not path:
-        path = "/"
-    return path
+    return http_request.method
```

Comment:
does this mean the return value will change here?

Reply:
Yes, the HTTP span name will change to conform with the OTel [HTTP semantic conventions](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#name). I plan to split out sem conv changes to a separate PR, and keep this focused on adding the native core tracing plumbing.